src/store/model/coolblue.ts

```diff
@@ -0,0 +1,72 @@
+import {Store} from './store';
+
+export const Coolblue: Store = {
+	labels: {
+		inStock: {
+			container: '.product-order',
+			text: ['bestel snel', 'Morgen in huis']
+		},
+		outOfStock: {
+			container: '.product-order',
+			text: ['Binnenkort leverbaar', 'Tijdelijk uitverkocht']
+		}
+	},
+	links: [
+		{
+			brand: 'test:brand',
+			model: 'test:model',
+			series: 'test:series',
+			url: 'https://www.coolblue.nl/product/826844/'
+		},
+		{
+			brand: 'gigabyte',
+			model: 'eagle oc',
+			series: '3080',
+			url: 'https://www.coolblue.nl/product/868737/gigabyte-geforce-rtx-3080-eagle-oc-10g.html'
```

Comment:
```suggestion
			url: 'https://www.coolblue.nl/product/868737'
```

Do you mind normalizing these URLs like the test model?

Thanks!

Reply:
Done! (I think, no idea how to work with Github, I usually just use Git by itself)